packages/dashboard/src/hooks/useRedPacketAddress.ts

```diff
@@ -0,0 +1,7 @@
+import { useRedPacketConstants } from '@masknet/web3-shared'
+
+export function useRedPacketAddress() {
+    const { HAPPY_RED_PACKET_ADDRESS_V2, HAPPY_RED_PACKET_ADDRESS_V3 } = useRedPacketConstants()
```

Comment:
@zhouhanseng Please review this.

Reply:
okay

Reply:
How about a transaction of `HAPPY_RED_PACKET_ADDRESS_V1`？I think this hook is unnecessary, you can use `useRedPacketConstants()` directly, then write a function to compare the address with V1 V2 and V3